backend/services/users.js

```diff
@@ -22,10 +22,20 @@ async function createUser(raw_user) {
   if (user) {
     throw ServiceError(409, "Email already taken");
   }
-  const raw_user_no_secret = { ...raw_user };
+
+  const pending_role = await Role.findOne({ name: "Pending" }).exec();
+  const raw_user_no_secret = {
+    ...raw_user,
+    phone: "(xxx)xxx-xxxx",
```

Comment:
Can we add a TODO comment, to update this in the future?

Reply:
done!